paddle/operators/detail/grpc_client.cc

```diff
@@ -63,9 +63,6 @@ bool RPCClient::AsyncGetVariable(const std::string& ep,
   sendrecv::VariableMessage req;
   req.set_varname(var_name);
 
-  auto* var = scope.FindVar(var_name);
```

Comment:
We need an issue explaining why would we remove these two lines.  Or, at least explain the reason in the description of this PR, so could we record the reasoning behind the change.

Reply:
Added the issue.